tests/ci/profile_handler.go

```diff
@@ -247,7 +258,58 @@ func PrepareKMSKey(profile *Profile, kmsName string, accountRolePrefix string, a
 	return kmsOutput.KeyARN, err
 }
 
-func PrepareRoute53() {}
+func PrepareRoute53() (string, error) {
+	s, err := EXE.NewDnsDomainService()
+	if err != nil {
+		return "", err
+	}
+	a := &EXE.DnsDomainArgs{}
+
+	err = s.Create(a)
+	if err != nil {
+		s.Destroy()
+		return "", err
+	}
+	output, err := s.Output()
+
+	return output.DnsDomainId, err
+}
+
+func PrepareSharedVpcPolicyAndHostedZone(profile *Profile,
```

Comment:
please keep independent from Profile structure so it can be reused elsewhere. Here just send the `region` instead

Reply:
Updated.